### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by orville-wright**
Signed-off-by: Orville Wright <orville.wright@yahoo.com>

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -25,14 +25,17 @@ _Note_: OpenEBS follows an aggressive release cadence with a new minor release e
...
-| 4.1  | DiskPool media aggregation mgmt LVM mode      | New DiskPool type (LVM Mediastore) of aggregated media devices managed under an integrated Linux LVM kernel    | Pri 1 /  Rel: (Q2 2024)    |
-| 6    | Local-PV Data-Engine(s) integrated + enabled  | Dynamically provision a persistent volume of LocalPV (non-replicated) type using non-SPDK blobstor for storage  | Pri 1 /  Rel: (Q1 2024)   |
-| 6.1  | Local-PV Hostpath enabled                     | Able to provision a persistent volume of Local-PV (non-replicated) using type : K8s Hostpath addressed storage  | Pri 2 /  Rel: (Q2 2024)   |
+| 4    | DiskPool Aggregate media mgmt SPDK mode       | Able to create, expand & manage virtual SPDK disks that are aggregates of multiple physical media devices      | Pri 1 /  Rel: (Q2 2024)    |
+| 4.1  | DiskPool storage media mgmt xVM mode          | New DiskPool type (xVM Mediastore) - Backend media devices are managed under LVM & ZFS kernel layers           | Pri 1 /  Rel: (Q2 2024)    |
+| 4.2  | DiskPool Choosable Replication mode           | New DiskPool enables user to select Replicated/Non-Replciated persona for any volume type (SPDK, LVM, ZFS)     | Pri 1 /  Rel: (Q2 2024)    |
```
Comment (tiagolobocastro) [inline ROADMAP.md:30]:
```suggestion
| 4.2  | DiskPool Choosable Replication mode           | New DiskPool enables user to select Replicated/Non-Replicated persona for any volume type (SPDK, LVM, ZFS)     | Pri 1 /  Rel: (Q2 2024)    |
```

File: ROADMAP.md [hunk 2]
```diff
@@ -25,14 +25,17 @@ _Note_: OpenEBS follows an aggressive release cadence with a new minor release e
...
+| 4.3  | DiskPool Chosable Data Protection mode        | New DiskPool enables RAID Levels 0,1,4,5,6,10 & Z,Z2,Z3 via DiskPool modes LVM & ZFS integrations              | Pri 1 /  Rel: (Q2 2024)    |
+| 5    | DiskPool Erasure Coded Data Protection mode   | New DiskPool enables Distributed Erasure Coding Data Protection as an alternative to RAID architecture         | Pri 1 /  Rel: (Q2 2024)    |
+| 6    | Local-PV Data-Engine integrated + enabled     | Dynamically provision a persistent volume of LocalPV (non-replicated) type using non-SPDK managed storage      | Pri 1 /  Rel: (Q1 2024)   |
+| 6.1  | Local-PV Hostpath integarted + enabled        | Able to provision a persistent volume of Local-PV (non-replicated) using type : K8s Hostpath addressed storage  | Pri 2 /  Rel: (Q2 2024)   |
```
Comment (tiagolobocastro) [inline ROADMAP.md:34]:
```suggestion
| 6.1  | Local-PV Hostpath integrated + enabled        | Able to provision a persistent volume of Local-PV (non-replicated) using type : K8s Hostpath addressed storage  | Pri 2 /  Rel: (Q2 2024)   |
```

**Review by tiagolobocastro — APPROVED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (orville-wright) [reply ROADMAP.md]: Thx

File: ROADMAP.md (hunk 2 above)
Comment (orville-wright) [reply ROADMAP.md]: Thx

**Comment by orville-wright** [comment]
Thx

**Pull request merged**